s3_policies/minio_create_bucket_entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Should this also be run on the main CDM MinIO instance, or is it intended only for local testing? Might be worth setting up a dedicated MinIO repo in the future.

Reply:
This is only for local testing. The reason the CTS is set up to run on the CDM instance is because you can't test running jobs without it

Reply:
> Might be worth setting up a dedicated MinIO repo

Why would we need a Minio repo?

Reply:
The current CDM MinIO instance was built using the scripts and JSON files from the cdm-jupyterhub repo, but it was configured manually. I'm thinking that if multiple repos start creating their own MinIO instances, it might make sense to consolidate everything into a single repo so that in case of re-deploy we don't forget of creating a specific user, bucket, etc.

Reply:
> CDM MinIO instance was built using the scripts and JSON files from the cdm-jupyterhub repo

That's not completely true - I also added some CTS policies manually

That being said, I see what you're saying - have a pre configured minio repo. I'd need to think about it - it seems like as we move forward we'd add more and more users and policies and things could get messy. For now it seems to be that each repo setting up just the users and policies they need is cleaner